### PR TITLE
Introduced the editor quick actions toolbar

### LINKED ISSUE #104: Toggle editor read-only state
Probably requries #38, but it would be nice to change the editor state from the Inspector.

I wanted to quickly make it disabled/enabled and for some reason, I forgot that `isReadOnly` is the name of the flag (I was searching for `isDisabled`... :facepalm: ).

### LINKED ISSUE #121: A simple button to get data
Writing `editor.getData()` every time I want to see the data output is annoying. And it gets even worse when the editor is not in the `window` scope and I have to log a DOM element, access `element.ckeditorInstance...` , and so on. I think the right place for global actions is somewhere in the red box

### PR DESCRIPTION
**Pull request by oleq**
Feature: Introduced editor quick actions toolbar (see #38) to log editor data, toggle editor read-only, and destroy editor. Closes #121. Closes #104.

---

## Notes

Under the hood, 

*   I refactored the `Button` component to be more versatile because the number of buttons increased and some need `isOn/isEnabled` states. 
*   Also, I changed the way icons are handled in the project (loaded from SVG instead of hardcoded base64), unified their size, and made them compatible with mentioned buttons.
*   Toolbar separators are spans with a class now for reusability because they appear in two different places.

### CODE REVIEW

**Review by psmyrek — APPROVED** [review]
LGTM 🎉 

I briefly scanned the CKEditor 5 Inspector code (as this was the first time I saw it), then I ran this PR and it works as expected.

The only one thing to consider is updating the screenshot of the Inspector in the docs after releasing new version: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/development-tools.html#ckeditor-5-inspector. I don't know if it appears elsewhere in the docs.

**Review by przemyslaw-zan — COMMENTED** [review]
Should the editor toolbar stay after destroying it? This happens for me for all three editors in the sample.
image

**Comment by oleq** [comment]
> Should the editor toolbar stay after destroying it? This happens for me for all three editors in the sample.

This is how decoupled document editor works. The integrator takes care of injecting the toolbar and removing it. It's just a sample so I didn't bother.

File: src/commands/commandinspector.js [hunk 1]
```diff
@@ -47,14 +50,14 @@ class CommandInspector extends Component {
...
+					icon={<PlayIcon />}
 					text="Execute command"
 					onClick={this.handleCommandExecuteButtonClick}
 				/>,
 				<Button
 					key="log"
-					type="log"
 					text="Log in console"
+					icon={<ConsoleIcon />}
 					onClick={this.handleCommandLogButtonClick}
```
[the remark this answers is not in the document]
Comment (oleq) [reply src/commands/commandinspector.js:61]: No idea why this happened :P

**Comment by oleq** [comment]
> The only one thing to consider is updating the screenshot of the Inspector in the docs after releasing new version: [https://ckeditor.com/docs/ckeditor5/latest/framework/guides/development-tools.html#ckeditor-5-inspector](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/development-tools.html#ckeditor-5-inspector). I don't know if it appears elsewhere in the docs.

README.md of the inspector on Github/Npm, that's for sure.

**Comment by pomek** [comment]
I extracted updating documentation to a new issue – https://github.com/ckeditor/ckeditor5-inspector/issues/133.

**Review by pomek — APPROVED** [review]
LGTM.

**Review by przemyslaw-zan — APPROVED** [review]
LGTM

File: src/commands/commandinspector.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (pomek) [reply src/commands/commandinspector.js]: Fixed in https://github.com/ckeditor/ckeditor5-inspector/pull/130/commits/c079a3a33700549de1d70a539ec374e5de00be2d.

**Pull request merged**